### ISSUE TITLE
CV Application Project: Bonus Section for the Project

### ISSUE THREAD

**Issue by mathdebate09** [TheOdinProject/curriculum#27825]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

I've repeatedly seen few submissions include a download button which could be done using [html2canvas](https://html2canvas.hertzen.com/getting-started).  If we could include this is a bonus task in CV Application project, I believe it might help.
I've personally found the bonus sections quite fun to work around with so I'd love if it's included in here too.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-react-new-cv-application

### (Optional) Discord Name

mathdebate09

### (Optional) Additional Comments

PS. - It's my first time opening an issue/attempting to contribute, i'd love to learn more

**Comment by JoshDevHub** [comment]
Thank you for making this issue @mathdebate09 

Gonna ping our React team to get opinions on this @TheOdinProject/react

**Comment by MaoShizhong** [comment]
I'm not sure we need to add that in, even as extra credit. The main focus of that project is to start getting used to using components and managing state, and a lot of people already have trouble handling the basic building blocks of React (which is understandable given it's the first project with it).

Admittedly, I'm personally more on the "reluctant" side regarding adding to/creating extra credit sections and would rather see projects focus on the core concepts they intend learners to focus on. If it's valuable enough of a step, it should be a mandatory spec point. If it's not valuable enough to be made as such, I feel like it probably shouldn't be included, as people may not be able to judge the scope well enough and end up wasting time trying to do something that's not part of the real point of the project.

This is also coming from my own experience, reflecting on how I spent more time trying to figure out how to structure and make component->PDF work for printing, than doing the core requirements. If I was dying to implement that feature, it'd be more suitable later on when I've more fundamental skills. But instead, I wasted time adding that feature poorly, and honestly it didn't really teach me a great deal.
